packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/TaskNavButtons.stories.js

```diff
@@ -0,0 +1,101 @@
+import { Provider } from 'mobx-react'
+import { Box } from 'grommet'
+import mockStore from '@test/mockStore'
+import mockWorkflow from './mocks/mockWorkflow'
+
+import TaskNavButtons from './TaskNavButtons'
+
+function ComponentDecorator(Story) {
+  return (
+    <Box
+      background={{ light: 'white', dark: 'dark-1' }}
+      pad='small'
+      width='25rem'
+    >
+      <Story />
+    </Box>
+  )
+}
+
+export default {
+  title: 'Tasks / Nav Buttons',
+  component: TaskNavButtons,
+  decorators: [ComponentDecorator]
+}
```

Comment:
These stories are great for reviewing the various UI states of TaskNavButtons. However, I chose not to import them into spec files for each button's Connector component because tests on the connectors involve "onClick" and inspecting changes in the MST.